samples/net/sockets/echo_client/prj.conf

```diff
@@ -40,7 +40,7 @@ CONFIG_NET_SHELL=y
 # For linux<->qemu connectivity, create a new conf file and swap the
 # addresses (so that peer address is ending to 2).
 CONFIG_NET_CONFIG_SETTINGS=y
-CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::2"
-CONFIG_NET_CONFIG_PEER_IPV6_ADDR="2001:db8::1"
-CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.2"
-CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1"
```

Comment:
Ah, I missed the point when you asked the IP address numbering. I think we need to leave these like this atm so that qemu<->qemu communication works ok.
When compiling the echo-client, we can override the default addresses so can do like this: `cmake .... -DCONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1" -DCONFIG_NET_CONFIG_PEER_IPV6_ADDR="2001:db8::2" -DCONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1" -DCONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"`

Reply:
Ok, let's keep the prj.conf and fix up the addresses assigned to the Docker image instead.